zingolib/Cargo.toml

```diff
@@ -7,6 +7,7 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [features]
+integration-tests = []
```

Comment:
what is the benefit of having a integration test feature over using cfg(test) ? if we use cfg(test) we wouldnt need to send a feature flag when running tests

Reply:
Good question.   The reason we can't use this attribute is that the *kind* of test that uses these methods, is an "integration" test, that means that the code-under-test is a compiled version of the lib, built in `release` mode.

Integration tests apply scenarios to a compiled artifact produced by a different crate to the integration test crate.

The `#[cfg(test)]` attribute selectively compiles code when tests are run *within* a particular crate.